common/dice/dice.go

```diff
@@ -46,7 +45,3 @@ func (dd *DeterministicDice) Roll(n int) int {
 	}
 	return dd.Intn(n)
 }
-
-func init() {
-	rand.Seed(time.Now().Unix())
-}
```

Comment:
This is not necessary anymore, as [after go 1.20, the seed will be randomly generated](https://pkg.go.dev/math/rand#Seed). And the runtime `fastrand64` might be used for lock-free random number generating.

see also: https://cs.opensource.google/go/go/+/refs/tags/go1.22.0:src/math/rand/rand.go;l=308-312